x/profile/types/genesis.go

```diff
@@ -97,9 +97,15 @@ func (gs GenesisState) ValidateCoordinators() error {
 			return errors.New("coordinator id should be lower or equal than the last id")
 		}
 		index := string(CoordinatorByAddressKey(elem.Address))
-		if _, ok := coordinatorByAddressIndexMap[index]; !ok {
+		_, found := coordinatorByAddressIndexMap[index]
+		if !found {
 			return errors.New("coordinator address not found for CoordinatorByAddress")
 		}
+
+		if !elem.Active {
```

Comment:
In the previous state, there was a 1-1 relationship between `Coordinator` and `CoordinatorByAddress` therefore we checked that all `Coordinator` had an associated `CoordinatorByAddress`

Now a `Coordinator` can eventually have no matched `CoordinatorByAddress` but if this is the case  the coordinator must be inactive.

Therefore we should rather have condition that allows inactive coordinators in the initial genesis

```
if !found && elem.Active {
	return errors.New("....")
}

if found && !elem.Active {
	return errors.New("....")
}
```